libr/anal/codemeta.c

```diff
@@ -41,10 +42,11 @@ R_API RCodeMeta *r_codemeta_clone(RCodeMeta *code) {
 }
 
 R_API RCodeMeta *r_codemeta_new(const char *code) {
+	R_RETURN_VAL_IF_FAIL (code, NULL);
 	RCodeMeta *r = R_NEW0 (RCodeMeta);
 	if (r) {
 		r->tree = r_crbtree_new (NULL);
-		r->code = code? strdup (code): NULL;
+		r->code = strdup (code);
```

Comment:
good catch this argument should never be null! there are many other codepaths that will fail otherwise. so lgtm!